app/src/main/java/org/kiwix/kiwixmobile/zim_manager/local_file_transfer/FileItem.java

```diff
@@ -0,0 +1,33 @@
+package org.kiwix.kiwixmobile.zim_manager.local_file_transfer;
+
+/**
+ * Helper class, part of the local file sharing module.
+ *
+ * Defines a file-item to represent the files being transferred.
+ * */
+public class FileItem {
+  public static final short TO_BE_SENT = -1;  // File yet to be sent
+  public static final short SENDING = 0;      // Being sent
+  public static final short SENT = +1;        // Successfully sent
+  public static final short ERROR = +2;       // Error encountered while transferring the file
```

Comment:
a unary plus is redundant here. I would also probably represent these statuses as an enum

Reply:
Adding signs was more of a cosmetic choice, for readablility {negative: yet to send, zero: sending, positive: transfer over (successful or otherwise)}

Apparently enums are now avoided because of memory issues, so I converted it using type-annotations - [Commit](https://github.com/kiwix/kiwix-android/pull/1276/commits/0c143300030415e96cf41c08142a478ee00acd5b)

Reply:
That stupid thing about enums on android is ridiculous, don't believe the docs. They are just defending the (awful) design of android where it probably paid to be hyper efficient when they had to run on phones in 2007. 
Enums are great and you pay a price of like 50 bytes. I very much recommend them

Reply:
Since the aim is to have the compiler enforcing use of only one of the constants as a valid value, can I leave it as such for now? I'll use enums whenever I'm writing constants for such a scenario in the future

Reply:
Sure